src/main/kotlin/calculator/Calculator.kt

```diff
@@ -1,26 +1,25 @@
 package calculator
 
+fun main() {
+    println(Calculator().input("1 + 2 + 3"))
+}
+
 class Calculator {
     fun input(expression: String?): Int {
-        checkExpressionNullOrEmpty(expression)
-        expression ?: return -1
+        requireNotNull(expression) { "null입니다." }
+        require(expression.isEmpty().not()) { "빈 공백 문자입니다." }
 
         val expressionContents: List<String> = expression.split(" ")
 
-        var result = 0
+        var result = expressionContents[0].toInt()
```

Comment:
```suggestion
        var result = expressionContents.first().toInt()
```
코틀린은 여러 확장 함수를 제공합니다.